### PR TITLE
TDD Dojo Demo

### PR DESCRIPTION
**Pull request by wpschaub**

### CODE REVIEW

**Comment by wpschaub** [comment]
Alex, here is the edited demo video and a teaser blog post. Please make changes, as needed.

**Review by wsbctechnicalblog — CHANGES_REQUESTED** [review]
Thank you Willy. On line 20 the typo "tht" should be "that".

Also, in the video the sound is ahead by a few seconds while the video lags. Could you fix that?

**Comment by wpschaub** [comment]
Line 20 was easy to fix. Will check the video created by Teams later this week.

**Review by erwsch1 — DISMISSED** [review]

**Comment by wsbctechnicalblog** [comment]
@wpschaub the video (https://youtu.be/7qcx3tEUSBE) is still lagging considerably behind the audio, making it hard to follow the presentation. Is there a way to synchronize the sound and the video?

**Comment by wpschaub** [comment]
I will check the video when I am back from vacation.

**Comment by wpschaub** [comment]
@wsbctechnicalblog if I add the pre-amble, or if I trim the start and the finish, YouTube processing seems to go off the rails. I have uploaded a new video, which is trimmed at the beginning only - audio looks better. I will try to trim ~4 seconds from the end next, but video feels usable at this point.
